.appveyor.yml

```diff
@@ -71,37 +71,37 @@ for:
       - image: Visual Studio 2013
   install:
   - cmd: >-
-      appveyor DownloadFile https://github.com/appleseedhq/appleseed-deps/releases/download/v17/appleseed-deps-stage-win64-vs120-v17.zip
+      appveyor DownloadFile https://github.com/appleseedhq/appleseed-deps/releases/download/v17/appleseed-deps-stage-win64-vs110-v17.zip
 
-      appveyor DownloadFile https://github.com/appleseedhq/appveyor-deps/releases/download/v1/boost_1_55_0_2013_py35.zip
+      appveyor DownloadFile https://www.dropbox.com/s/fwjzqc2m1a5euej/boost_1_55_0_vc11.zip?dl=1
 
-      7z x appleseed-deps-stage-win64-vs120-v17.zip -oappleseed-deps-stage-win64-vs120-v17
+      7z x appleseed-deps-stage-win64-vs110-v17.zip -oappleseed-deps-stage-win64-vs110-v17
 
-      7z x boost_1_55_0_2013_py35.zip -oboost_1_55_0_2013_py35
-
-      rm appleseed-deps-stage-win64-vs120-v17.zip
-
-      rm boost_1_55_0_2013_py35.zip
+      7z x boost_1_55_0_vc11.zip -oboost_1_55_0_vc11
```

Comment:
I would prefer for the folder to be called `boost_1_55_0`, that's more standard.